packages/server/test/unit/socket_spec.js

```diff
@@ -47,7 +47,10 @@ describe('lib/socket', () => {
   afterEach(function () {
     Fixtures.remove()
 
-    return this.server.close()
+    return Promise.join(
+      this.server.close(),
+      ctx.actions.project.clearCurrentProject(),
+    )
```

Comment:
Why not use the standard Promise.all?

Reply:
Good catch. I was copying pasting from elsewhere. I missed a couple as well.